File: HarshKesarwani/target.cpp
#include <iostream>
#include <vector>
using namespace std;

int main()
{
    int size;
    cout << "Enter size" << endl;
    cin >> size;
    int nums[size], index[size];
    vector<int> target(size);
    cout << "Input Numbers and Index" << endl;
    for (int x = 0; x < size; x++)
    {
        cout << "Enter number : ";
        cin >> nums[x];
        cout << "Enter index : ";
        cin >> index[x];
    }
    cout << endl;

    int s = size;
    int in = 0;
    while (s != 0)
    {
        int i = index[in];
        int n = nums[in];
        target.insert(target.begin() + i, n);
        in++;
        s--;
    }

    for (int x = 0; x < size; x++)
    {
        if (x == 0)
        {
            cout << "[" << target[x] << " , ";
        }
        else if (x == size - 1)
        {
            cout << target[x] << "]";
        }
        else
        {
            cout << target[x] << " , ";
        }
    }
    return 0;
}

File: Ananya Chandra/target.py
# indices of two numbers of array whose sum gives target number. 
try:
    arr= []
    print(" Enter the integer inputs and type 'stop' when you are done\n" )
    while True:
        arr.append(int(input()))

except:# if the input is not-integer, just continue to the next step
    target= int(input(" Enter the target integer\n"))
    l=len(arr)
    for i in range(0,l-1):
        for j in range(i+1,l):
            if (arr[i]+arr[j])==target:
                print(i,",",j)
